python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -78,16 +94,20 @@ def __init__(
         self,
         task_index: int,
         streaming_gen: ObjectRefGenerator,
-        output_ready_callback: Callable[[RefBundle], None],
+        output_ready_callback: Callable[
+            [RefBundle, Union[type, "PandasBlockSchema", "pa.lib.Schema"]], None
```

Comment:
Let's create internal `Schema` alias for that to avoid the need to specify unions everywhere